lib/ansible/modules/cloud/azure/azure_rm_trafficmanagerendpoint_facts.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Hai Cao, <t-haicao@microsoft.com>, Yunge Zhu <yungez@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_trafficmanagerendpoint_facts
+
+version_added: "2.7"
+
+short_description: Get Azure Traffic Manager endpoint facts
+
+description:
+    - Get facts for a specific Traffic Manager endpoints or all endpoints  in a Traffic Manager profile
+
+options:
+    name:
+        description:
+            - Limit results to a specific Traffic Manager endpoint.
+    resource_group:
+        description:
+            - The resource group to search for the desired Traffic Manager profile
+        required: True
+    profile_name:
+        description:
+            - Name of Traffic Manager Profile
+        required: True
+    type:
+        description:
+            - Type of endpoint.
+        choices:
+            - azureEndpoints
```

Comment:
See my point in the non facts module around snake_case.

Reply:
fixed